src/utils/recap.service.in

```diff
@@ -5,5 +5,6 @@ After=network-online.target
 Wants=network-online.target
 
 [Service]
+Environment=BINDIR=/usr/local/bin
 Type=oneshot
-ExecStart=@BINDIR@/recap
+ExecStart=/bin/sh -c "${BINDIR}/recap"
```

Comment:
Any particular reason to spawn `recap` into a shell?

Reply:
Yep, responded above